samples/shields/x_nucleo_iks01a1/README.rst

```diff
@@ -0,0 +1,57 @@
+.. _x-nucleo-iks01a1-sample:
```

Comment:
You'll also need to add a line in ``samples/samples.rst`` to include ``shields/*`` to get his new sample tree included in the generated docs.

Reply:
Indeed, had missed that, thanks!